.eslintrc.js

```diff
@@ -5,20 +5,36 @@ module.exports = {
     browser: true,
     jest: true,
   },
-  extends: ['airbnb-base', 'plugin:flowtype/recommended', 'prettier', 'plugin:prettier/recommended'],
+  extends: [
+    'airbnb-typescript/base',
+    'airbnb/hooks',
```

Comment:
Did you sneak React hooks into the project too? 😄 
```suggestion
```

Reply:
Lol, force of habit!  I'll remove.